operations/rack_resiliency/rack_resiliency.md

```diff
@@ -0,0 +1,484 @@
+# Rack Resiliency (RR)
+
+* [Introduction](#introduction)
+* [RR solution overview](#rr-solution-overview)
+* [Steps to achieve RR](#steps-to-achieve-RR)
+* [Verification of RR](#verification-of-RR)
+* [Manage RR via CLI](#Manage-RR-via-CLI)
+* [Manage Critical Services](#manage-critical-services)
+* [Glossary](#glossary)
+
+## Introduction
+
+HPE Cray EX systems are engineered to ensure services critical to the execution of user jobs remain resilient, even if management nodes (Master, Worker, and Storage nodes) fail, provided that the minimum quorum of nodes required for high availability (HA) is maintained.
+
+Despite these methods of software resiliency, rack-level failures can cause service disruptions if management nodes running critical services are concentrated within a single rack. The loss of critical management services can occur when multiple nodes located in the same rack fail, resulting in the loss of the quorum necessary for HA. To mitigate this, we need to physically place management nodes across racks, then ensure CSM service replicas are evenly instantiated across those nodes. However, `k8s' does not inherently recognize the physical distribution of nodes across different racks – it needs to be configured using its zone-awareness features.
+
+To address above issues, Rack Resiliency feature is introduced to provide management rack level resiliency by providing methods and functionality required to maintain the high availability of critical management services due to unplanned outages. This feature will ensure that CSM can tolerate failures of racks and prevent a system wide outage, where such outage encompasses the inability for completing current jobs or scheduling new ones. This feature doesn't address planned downtime of racks for the current release (CSM 1.7.0).

```

Comment:
The linter is probably complaining about how long this line is anyway, but even if we had no linter, it's really in your best interest to keep the lines shorter. As you can see in this review, there are already a bunch of suggestions for changes to this paragraph. Because it's all 1 line, it'll be a lot harder for you to make sure they are all addressed, and it will prevent you from simply being able to commit the comments that have suggestions, since they will cause merge conflicts after the first one.